es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```

Comment:
Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)

Reply:
![es6-classes1](https://user-images.githubusercontent.com/64106926/227423691-6dd472d9-20cb-41c4-a4da-4b1097948767.png)